scripts/oneoff/inject-framework-dates.py

```diff
@@ -26,20 +26,23 @@
         'applicationsCloseAtUTC': '2016-01-19T15:00:00Z',
         'intentionToAwardAtUTC': '2016-02-18T12:00:00Z',
         'frameworkLiveAtUTC': '2016-03-03T12:00:00Z',
+        'frameworkExpiresAtUTC': '1970-01-01T00:00:00Z',
     },
     'digital-outcomes-and-specialists-2': {
         'clarificationsCloseAtUTC': '2016-12-01T17:00:00Z',
         'clarificationsPublishAtUTC': '2016-12-07T17:00:00Z',
         'applicationsCloseAtUTC': '2016-12-14T17:00:00Z',
         'intentionToAwardAtUTC': '2017-01-16T12:00:00Z',
         'frameworkLiveAtUTC': '2017-01-27T12:00:00Z',
+        'frameworkExpiresAtUTC': '2018-01-29T12:00:00Z',
```

Comment:
DOS2 already expired?

Reply:
This is the date currently stored on the DOS2 framework.frameworkAgreementDetails.frameworkEndDate (i.e. what was put on the original framework agreements). It may be that we should update it, but I don't know what the 'correct' expiry date is. So this feels OK for the moment.

Reply:
Ah ok. Did DOS2 get extended? If so maybe this was the original date. If that's the case maybe this should be updated (not now, just in general if this happens).

Reply:
Yeah it got extended. We should try to find out what the current expected end date is.